base/setup/reactos/lang/fi-FI.rc

```diff
@@ -17,210 +17,210 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
 CAPTION "ReactOS Setup"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    AUTORADIOBUTTON "Install ReactOS", IDC_INSTALL, 7, 7, 277, 10, WS_GROUP | WS_TABSTOP
-    LTEXT "Install a new copy of ReactOS. This option does not keep your files, settings and programs. You can make changes to disks and partitions.", IDC_INSTALLTEXT, 18, 18, 269, 25, NOT WS_GROUP
-    AUTORADIOBUTTON "Update or repair an installed ReactOS", IDC_UPDATE, 7, 50, 277, 10
-    LTEXT "Update or repair an installed copy of ReactOS. This option keeps your files, settings and programs. This option is only available if ReactOS is already installed on this computer.", IDC_UPDATETEXT, 18, 61, 269, 30, NOT WS_GROUP
-    LTEXT "Click Next to setup the devices.", IDC_STATIC, 7, 128, 297, 8
+    AUTORADIOBUTTON "Asenna ReactOS", IDC_INSTALL, 7, 7, 277, 10, WS_GROUP | WS_TABSTOP
+    LTEXT "Asenna uusi ReactOS kopio. Tämä vaihtoehto ei säilytä tiedostojasi, asetuksiasi ja sovelluksia. Voit tehdä muutoksia levyihin ja osiohin.", IDC_INSTALLTEXT, 18, 18, 269, 25, NOT WS_GROUP
+    AUTORADIOBUTTON "Päivitä tai korjaa ReactOS-asennus", IDC_UPDATE, 7, 50, 277, 10
+    LTEXT "Päivitä tai korjaa ReactOS-asennus. Tämä vaihtoehto säilyttää tiedostot, asetukset ja sovellukset. Tämä vaihtoehto on vain käytettävissä, jos ReactOS on asennettu tietokoneelle.", IDC_UPDATETEXT, 18, 61, 269, 30, NOT WS_GROUP
+    LTEXT "Paina Seuraava jatkaaksesi.", IDC_STATIC, 7, 128, 297, 8
 END
 
 IDD_UPDATEREPAIRPAGE DIALOGEX 0, 0, 317, 143
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
 CAPTION "ReactOS Setup"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT       "The ReactOS Setup can upgrade one of the available ReactOS installations listed below, or, if a ReactOS installation is damaged, the Setup program can attempt to repair it.", IDC_STATIC, 6, 6, 300, 18
+    LTEXT       "ReactOS-asennin voi päivittää alhaalla olevia asennuksia, tai jos asennus on vahingoittunut, niin asennin voi yrittää korjata sen.", IDC_STATIC, 6, 6, 300, 18
     CONTROL     "", IDC_NTOSLIST, "SysListView32", LVS_REPORT | LVS_SINGLESEL | LVS_SHOWSELALWAYS | LVS_ALIGNLEFT | WS_BORDER | WS_TABSTOP, 6, 30, 303, 90
-    PUSHBUTTON  "&Do not upgrade", IDC_SKIPUPGRADE, 230, 128, 80, 14
-    LTEXT       "Click Next to upgrade the selected OS installation, or on 'Do not upgrade' to continue a new installation without upgrading.", IDC_STATIC, 7, 124, 222, 16
+    PUSHBUTTON  "&Älä päivitä", IDC_SKIPUPGRADE, 230, 128, 80, 14
+    LTEXT       "Paina Seuraava päivittääksesi OS-asennuksen tai "Älä päivitä" jatkaaksesi ilman päivitystä.", IDC_STATIC, 7, 124, 222, 16
```

Comment:
```suggestion
    LTEXT       "Paina Seuraava päivittääksesi OS-asennuksen tai ""Älä päivitä"" jatkaaksesi ilman päivitystä.", IDC_STATIC, 7, 124, 222, 16
```
Missing was the escaping of the double quotes. Alternatively you might consider using single-quotes as was done in the English translation, unless this is something completely unnatural in Finnish.

Reply:
Yes, we could either use `""` to get a single `"` displayed later, like Hermes suggested. Or we could use a single `'` which gives a bit easier readable code. Benjamin should pick what feels most natural for finnish.

Reply:
I changed `"` to `""`. `"` is mainly used in finnish.